tests/utils/utils.go

```diff
@@ -196,7 +196,7 @@ func RedeemRelayerRewardsAndConfirm(
 	updatedRewardAmount, err :=
 		subnet.TeleporterMessenger.CheckRelayerRewardAmount(&bind.CallOpts{}, relayerAddress, feeTokenAddress)
 	Expect(err).Should(BeNil())
-	Expect(updatedRewardAmount).Should(Equal(big.NewInt(0)))
+	Expect(updatedRewardAmount.Cmp(big.NewInt(0))).Should(Equal(0))
```

Comment:
FYI @geoff-vball added a function in https://github.com/ava-labs/teleporter/pull/40 that does this: https://github.com/ava-labs/teleporter/blob/4a6059a85f4e4c601d511a077ea29d6411a3c983/tests/utils/utils.go#L538